pac_darwin.go

```diff
@@ -17,6 +19,6 @@ func ensureElevatedOnDarwin(be *byteexec.Exec, helperFullPath string, prompt str
 	if s.Mode&syscall.S_ISUID > 0 && s.Uid == 0 && s.Gid == 0 {
 		return
 	}
-	cmd := be.Command("elevate", prompt, iconFullPath)
+	cmd := elevate.PromptWithIcon(prompt, iconFullPath, be.Filename, "setuid")
```

Comment:
There were some bugs in your elevate code (I'll explain these in the other PR) and I don't see why we should maintain our own version of that code when [cocaosudo](https://github.com/performantdesign/cocoasudo) already has working code that's in production.